src/client/components/editor/caption.js

```diff
@@ -0,0 +1,38 @@
+import h from 'react-hyperscript';
+import { Component } from 'react';
+import _ from 'lodash';
+import TextareaAutosize from 'react-autosize-textarea';
+import { makeClassList } from '../../dom';
+
+export class Caption extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = ({ caption: props.document.caption() });
+  }
+
+  render(){
+    const { document } = this.props;
+
+    return h('div.editor-caption', [
+      document.editable() ? 
+        h(TextareaAutosize, {
+          className: makeClassList({
+            'editor-caption-textarea': true
+          }),
+          value: this.state.caption,
+          placeholder: `Figure caption (e.g. A phosphorylates B in breast cancer)`,
```

Comment:
This is where the example text lives.  Feel free to directly change it here with an improved placeholder.

Reply:
Describe the context of this article (e.g. The set of interactions connecting apoptosis to nuclear function in human breast cancer)